open-build-service-api/tests/integration.rs

```diff
@@ -47,11 +47,12 @@ fn create_authenticated_client(mock: ObsMock) -> Client {
 #[tokio::test]
 async fn test_package_list() {
     let mock = start_mock().await;
+    mock.add_project(test_project());
```

Comment:
Fwiw i'd have had add_project return a MockProject on which you can call e.g. add_package_revision and to which you can directly add repositories rather they keying of the path all the time ;)